Rakefile

```diff
@@ -29,6 +29,8 @@ DESIRED_COMPONENTS = [
   'Validation',
   'Wallet',
 ].freeze
+COMPONENTS = DESIRED_COMPONENTS.map(&:downcase).freeze
+UNDESIRED_PR_TITLE_WORDS = %w(- _).freeze
```

Comment:
Can you comment what this is matching on/doing?

Reply:
Sure. While testing different PRs after adding the logging, I came across ones that began with "- (rest of title)" or "_ (rest of title)" after the prefix was removed.

Any unwanted words at the start of the post PR title like that (in this case, single character words) can be added to this array to be removed.

`%w(- _)`

is array shorthand for

`["-", "_"]`

Reply:
Thanks. Perhaps you could add a comment, including example PR titles that get fixed by this.

Reply:
Good idea. Here's one:
```
rake -- posts:new -h a -p 16729
Date set to next Wednesday: 2019-12-25
GitHub PR title:  "[GUI] - Reintroduce menu icons"
Parsed PR title:  "Reintroduce menu icons"
GitHub PR labels: ""
Parsed PR labels: ""
New file _posts/2019-12-25-#16729.md created successfully.
```

Reply:
Done